pkg/loghttp/labels.go

```diff
@@ -89,20 +89,31 @@ func ParseLabelQuery(r *http.Request) (*logproto.LabelRequest, error) {
 }
 
 func ParseDetectedLabelsQuery(r *http.Request) (*logproto.DetectedLabelsRequest, error) {
-	var err error
-
 	start, end, err := bounds(r)
 	if err != nil {
 		return nil, err
 	}
 
+	minCardinality, err := parseInt(r.Form.Get("minCardinality"), 0)
+	if err != nil {
+		return nil, err
+	}
+
+	maxCardinality, err := parseInt(r.Form.Get("maxCardinality"), 0)
```

Comment:
Don't know where this is usually done, but can't we do this check also directly here?
```go
if maxCard == 0 {
  maxCard = uint64(math.Inf(1))
}
```